standalone-metastore/metastore-common/pom.xml

```diff
@@ -25,6 +25,11 @@
     <standalone.metastore.path.to.root>..</standalone.metastore.path.to.root>
   </properties>
   <dependencies>
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-jexl3</artifactId>
+      <version>3.3</version>
```

Comment:
Can we replace this with place holder and define the actual version standalone-metastore/pom.xml?

Reply:
You mean declare/use a ${jexl.version} ?